frontend/src/metabase/home/components/HomeContent/HomeContent.tsx

```diff
@@ -14,9 +15,13 @@ import { HomeRecentSection } from "../HomeRecentSection";
 import { HomeXraySection } from "../HomeXraySection";
 import { getIsXrayEnabled } from "../../selectors";
 import { isWithinWeeks } from "../../utils";
+import { shouldSHowEmbedHomepage } from "../EmbedMinimalHomepage/util";
```

Comment:
I'm not sure this util should be imported from a file under another component. Maybe it would be better to move the util to somewhere else more generic e.g. `frontend/src/metabase/home/utils.ts`.

Also
```suggestion
import { shouldShowEmbedHomepage } from "../EmbedMinimalHomepage/util";
```

Reply:
Moved all the utils to `/home/utils.ts` 👍  e97cde235fa6e0c7b30a95af1bb2999fe6bddee5